maint/extractcvars.in

```diff
@@ -196,6 +217,9 @@ print OUTPUT_H <<EOT;
 /* Arbitrary, simplifies interaction with external interfaces like MPI_T_ */
 #define ${uc_ns}_MAX_STRLEN (384)
 
+/* Shortens enum value comparisons */
+#define ${uc_ns}_ENUM(A, a) (${uc_ns}_ ## A == ${uc_ns}_ ## A ## _ ## a)
```

Comment:
The macro name is a little confusing. Can it be    ${uc_ns}_ENUM_EQ or _IS  to suggest it is a boolean expression?

Reply:
I like `MPIR_CVAR_ENUM_IS`. 

FYI, I added this macro because the name gets so long that it can't fit in one line, makes reading the code much more difficult.